themes/clinical-lowlight-theme/clinical-lowlight-theme.scss

```diff
@@ -0,0 +1,156 @@
+:global {
```

Comment:
Should this be converted or will there be a follow up pr to convert these to be theme imports?

Reply:
If there's any modernizing to do for terra-application-navigation, I'd prefer to come back around with a follow-up PR.